src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -839,17 +845,16 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
     <!--
       Log a warning if:
-        1. $(EnableBaseIntermediateOutputPathMismatchWarning) is 'true'
-        2. $(BaseIntermediateOutputPath) was set in the body of a project after its default value was set in Microsoft.Common.props
-        3. $(BaseIntermediateOutputPath) is not the same as $(MSBuildProjectExtensionsPath)
+        1. $(EnableBuildDirMismatchWarning) is 'true'
+        2. $(BuildDir) was set in the body of a project after its default value was set in Microsoft.Common.props
+        3. $(BuildDir) is not the same as $(MSBuildProjectExtensionsPath)
 
-      Similar to the error above, there are cases when users set $(BaseIntermediateOutputPath) in the body of their project and things build but only by coincidence.
-      MSBuild does not know if $(BaseIntermediateOutputPath) changing would cause problems so tools like NuGet must set $(EnableBaseIntermediateOutputPathMismatchWarning)
-      to 'true'.
+      Similar to the error above, there are cases when users set $(BuildDir) in the body of their project and things build but only by coincidence.
+      MSBuild does not know if $(BuildDir) changing would cause problems so tools like NuGet must set $(EnableBuildDirMismatchWarning) to 'true'.
     -->
-    <Warning Condition=" '$(EnableBaseIntermediateOutputPathMismatchWarning)' == 'true' And '$(_InitialBaseIntermediateOutputPath)' != '$(BaseIntermediateOutputPath)' And '$(BaseIntermediateOutputPath)' != '$(MSBuildProjectExtensionsPath)' "
+    <Warning Condition=" '$(EnableBuildDirMismatchWarning)' == 'true' And '$(_InitialBuildDir)' != '$(BuildDir)' And '$(BuildDir)' != '$(MSBuildProjectExtensionsPath)' "
```

Comment:
Where is _InitialBuildDir set?

Reply:
Line 41 of src/Tasks/Microsoft.Common.props